src/components/AddTodo/AddTodo.tsx

```diff
@@ -0,0 +1,22 @@
+export function AddTodo() {
```

Comment:
AddTodo (with verb) sounds like a function and not a component. Rename it to TodoForm or TodoInput or even Header (you return header here)